fhevm/instructions_test.go

```diff
@@ -238,6 +238,12 @@ func newTestEVMEnvironment() *MockEVMEnvironment {
 	return &MockEVMEnvironment{fhevmData: &fhevmData, stateDb: state, commit: true, fhevmParams: DefaultFhevmParams()}
 }
 
+func newTestEVMEnvironmentWithEthCall() *MockEVMEnvironment {
+	mockEVM := newTestEVMEnvironment()
+	mockEVM.ethCall = true
+	return mockEVM
+}
+
```

Comment:
Not sure it's super worth it to create a separate function in a separate file just for one instance.

I would rather add the line `mockEVM.ethCall = true` inside tee_crypto_test.go